README.md

```diff
@@ -84,27 +84,31 @@ const init = async () => {
 };
 ```
 
-- `additionalHeaders`: (object) request headers that are passed on top of the existing ones.
+- `fetchOptions`: (object) options for the fetch configuration that will be merged with the internal options.
 
-`additionalHeaders` can also be provided at local level, in that case it will override the global headers.
+`fetchOptions` can also be provided at local level, in that case it will override the global configuration.
 
 ```js
 import API from '@teamleader/api';
 
 const { users } = API({
   getAccessToken: () => 'thisisatoken',
-  additionalHeaders: {
-    'x-tl-feature-flags': 'core.some-feature-flag=true',
+  fetchOptions: {
+    headers: {
+      'x-tl-feature-flags': 'core.some-feature-flag=true',
```

Comment:
Could be a good oportunity to use a different example header, as we dont want to give away that integrators can do this?

Reply:
Hmm, yeah but then I'd like to pick it up in a bigger documentation overhaul initiative